client/core/core.go

```diff
@@ -210,9 +211,11 @@ func (dc *dexConnection) parseMatches(msgMatches []*msgjson.Match) (map[order.Or
 			errs = append(errs, err.Error())
 			continue
 		}
-		err = dc.acct.checkSig(sigMsg, msgMatch.Sig)
-		if err != nil {
-			return nil, nil, fmt.Errorf("parseMatches: %v", err)
+		if checkSigs {
```

Comment:
This is necessary because the matches in the `connect` response aren't signed.

Reply:
Maybe we could have the server sign them, which would also serve as an early server pubkey verification: https://github.com/decred/dcrdex/issues/273  For future work though.